src/app/(fullscreen)/event/success/_components/client.tsx

```
@@ -82,10 +88,10 @@ export default function Client() {
           iconType="alert"
           title="이벤트 생성을 취소할까요?"
           children="지금까지 작성한 내용은 저장되지 않아요"
-          confirmText="생성 취소하기"
-          cancelText="아니오"
-          onConfirm={handleConfirmExit}
-          onCancel={handleCancelExit}
+          confirmText="돌아가기"
+          cancelText="생성 취소"
+          onCancel={handleConfirmExit}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)